packages/components/button/_button.scss

```
@@ -142,6 +141,25 @@ $button-shadow-size: 4px;
     background-color: $ofh-secondary-button-color;
   }
 
+  &:visited {
+    color: white;
+  }
+
```

You could use the colour variables in `packages/core/settings/_colours.scss`:

```suggestion
  &:visited {
    color: $color_ofh-white;
  }
```

(also extra line)